### PR TITLE
Mention important additional parameters for engine config in README

### PR DESCRIPTION
**Pull request by zaycev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,7 +88,9 @@ tokenizer = AutoTokenizer.from_pretrained("meta-llama/Llama-2-7b-chat-hf", paddi
 
 model = AutoModelForCausalLM.from_pretrained(
   "meta-llama/Llama-2-7b-chat-hf",
-+ use_fp8=True,  
++ use_fp8=True,
++ max_prompt_length=8_000,
```
Comment (fxmarty) [inline README.md:92]:
```suggestion
+ max_prompt_length=8000,
```

Comment (fxmarty) [inline README.md:92]: `max_output_length` could be added as well (e.g. Yi 7B OOMs in case left as default as it may have a long context length)

Comment (zaycev) [reply README.md]: @fxmarty the default is `128`, which isn't really reasonable for most practical cases.

Also, regarding max output length, looks like it depends on both max input and embeddings size:

https://github.com/huggingface/optimum-nvidia/blob/main/src/optimum/nvidia/builder/config.py#L82
```python
        max_batch_size = additional_params.pop("max_batch_size", 1)
        max_prompt_length = additional_params.pop("max_prompt_length", 128)
        max_new_tokens = (
            additional_params.pop("max_output_length", config.max_position_embeddings)
            - max_prompt_length
        )

        if max_new_tokens < 1:
```

Seems like it has to be at least `max_new_tokens + max_prompt_length`. I think it's also worth mentioning in README

Comment (mfuntowicz) [reply README.md]: I agree the `128` might be too small for practical usage, but 8000 and 16000 are above what's supported on 4090.

We certainly like to have something that can work in a vast majority of scenarios, so maybe 512/1024 or 1024/2048

Comment (zaycev) [reply README.md]: Changed to:
```diff
+ max_prompt_length=1024,
+ max_output_length=2048, # Must be at least size of max_prompt_length + max_new_tokens
+ max_batch_size=8,
```

**Review by mfuntowicz — APPROVED** [review]
LGTM! Thanks @zaycev

**Comment by zaycev** [comment]
@mfuntowicz  Thanks for looking! Wonder if anyone with correct permissions can merge this?

**Comment by mfuntowicz** [comment]
Thanks a lot @zaycev your contribution, looking forward for the next one 🤗

**Pull request merged**